RealmModelGenerator/JavaContentGenerator.swift

```diff
@@ -0,0 +1,208 @@
+//
+//  FileContentGenerator.swift
+//  RealmModelGenerator
+//
+//  Created by Zhaolong Zhong on 3/6/16.
+//  Copyright © 2016 QuarkWorks. All rights reserved.
+//
+
+import Foundation
+import AddressBook
+
+class JavaContentGenerator {
+    static let TAG = NSStringFromClass(JavaContentGenerator)
+    
+    var content = ""
+    var entity: Entity
+    
+    init(entity: Entity) {
+        self.entity = entity
+    }
+    
+    func getContent()->String
+    {
+        appendHeader()
+        appendRealmKeys()
+        appendAttributes()
+        appendGettersAndSetters()
+        
+        return content
+    }
+    
+    //MARK: Append header
+    func appendHeader()
+    {
+        appendHeaderComments()
+        
+        // Static import
+        content += "import io.realm.*;\nimport io.realm.annotations.*;\nimport java.util.Date;\n"
```

Comment:
should have an if check to se if java.util.Date should be included based if there are any attributes that are type Date